percona-xtradb-cluster-8.0-backup/get-pxc-state

```diff
@@ -30,6 +30,7 @@ while read -ra LINE; do
     ONLINE=$(echo "$STATUS" | grep wsrep_connected | awk '{print$2}')
     STATE=$(echo "$STATUS" | grep wsrep_local_state_comment | awk '{print$2}')
     CLUSTER_STATUS=$(echo "$STATUS" | grep wsrep_cluster_status | awk '{print$2}')
+    CLUSTER_SIZE=$(echo "$STATUS" | grep wsrep_cluster_size | awk '{print$2}')
 
-    echo node:$LINE:wsrep_ready:$READY:wsrep_connected:$ONLINE:wsrep_local_state_comment:$STATE:wsrep_cluster_status:$CLUSTER_STATUS
+    echo node:$LINE:wsrep_ready:$READY:wsrep_connected:$ONLINE:wsrep_local_state_comment:$STATE:wsrep_cluster_status:$CLUSTER_STATUS:wsrep_cluster_size:$CLUSTER_SIZE
```

Comment:
what cluster size value we will get here if we down scale 3-members cluster to 1-member and run backup immediately?

Reply:
```
++ FIRST_NODE=cluster1-pxc-0
++ SKIP_FIRST_POD='|'
++ ((  1 > 1  ))
++ peer-list -on-start=/usr/bin/get-pxc-state -service=cluster1-pxc
++ grep wsrep_ready:ON:wsrep_connected:ON:wsrep_local_state_comment:Synced:wsrep_cluster_status:Primary
++ sort
++ grep -v '|'
++ tail -1
++ cut -d . -f 1
++ cut -d : -f 2
+ local NODE_NAME=cluster1-pxc-0
+ '[' -z cluster1-pxc-0 ']'
+ set +o errexit
+ echo '[INFO] garbd was started'
+ garbd --address 'gcomm://cluster1-pxc-0.cluster1-pxc?gmcast.listen_addr=tcp://0.0.0.0:4567' --donor cluster1-pxc-0 --group cluster1-pxc --options '' --sst xtrabackup-v2:10.12.8.42:4444/xtrabackup_sst//1 --recv-script=/usr/bin/run_backup.sh
```
@delgod  as I can see it  works